nix/packages.nix

```diff
@@ -14,11 +14,31 @@
       ];
       pkgs = config._module.args.pkgs.extend (lib.composeManyExtensions overlays);
       flake = pkgs.ytxp-plutarch.flake { };
+      combine-haddock = pkgs.fetchurl {
+        url = "https://raw.githubusercontent.com/Plutonomicon/plutarch-plutus/f78b70eca8841a4a009cb6791a59c999cbc68745/nix/combine-haddock.nix";
```

Comment:
Not a blocker: but I assume this would not fail if we update the plutarch version [here](https://github.com/mlabs-haskell/ytxp-lib/blob/master/ytxp-plutarch/cabal.project#L15) and forget this one, is there any way to enforce this, or even better avoid duplicating the SHA?